tools/torch_quant/torch_quant/module.py

```diff
@@ -22,11 +22,101 @@ class TracePair(NamedTuple):
 
 
 # eval priority: exclude > include, name > type
-class ModuleFilter(NamedTuple):
-    include_names: Optional[List[str]] = None
-    include_types: Optional[List[Type[nn.Module]]] = None
-    exclude_names: Optional[List[str]] = None
-    exclude_types: Optional[List[Type[nn.Module]]] = None
+class ModuleFilter:
+    def __init__(
+        self,
+        include_names: Optional[List[str]] = None,
+        include_classes: Optional[List[Type[nn.Module]]] = None,
+        include_op_types: Optional[List[Type[nn.Module]]] = None,
+        exclude_names: Optional[List[str]] = None,
+        exclude_classes: Optional[List[Type[nn.Module]]] = None,
+        exclude_op_types: Optional[List[Type[nn.Module]]] = None,
```

Comment:
add a TODO: better names for *_classes, *_op_types arguments

Reply:
Done